src/api/app/models/relationship.rb

```diff
@@ -167,7 +167,7 @@ def allowed_user
     raise NotFoundError, "Couldn't find user #{user.login}" if user && user.is_nobody?
   end
 
-  def create_event_after_create
+  def create_relationship_creation_event
```

Comment:
IMO `create_relationship_create_event` is better

Reply:
`after_create: create_relationship_create_event` VS `after_create: create_relationship_creation_event` :thinking: